core/css/decanter.css

```
@@ -8740,16 +8740,13 @@ a {
   margin-top: 0;
   margin-right: 0;
   margin-bottom: 0;
-  background-color: transparent;
+  background: url("../img/icon-search.svg") no-repeat 0 0;
```

```suggestion
  background: url("#{$image-path}/icon-search.svg") no-repeat 0 0;
```
I think this should refer to the $image-path variable.

---

This is the rendered CSS. The SCSS does have the correct variable.

---

+1 Was just typing that. It looks like the SCSS is using the variable correctly.

---

Oops, my bad. Sorry 'bout that.